sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -202,7 +202,7 @@ function allFoldersForPackages(
                 // .devDependencies or or .peerDependencies.  These are not what are considered part
                 // of the final runtime configuration of the app and should not be uploaded.
                 const referencedPackages = new Set<string>(includedPackages);
-                if (root.package && root.package.dependencies) {
+                if (root?.package.dependencies) {
```

Comment:
Shouldn't this be `root.package?.dependencies`?

Reply:
Uhhh... yes.